src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -47,7 +47,7 @@ public function denormalize($data, $class, $format = null, array $context = arra
         $normalizedData = $this->prepareForDenormalization($data);
 
         $reflectionClass = new \ReflectionClass($class);
-        $object = $this->instantiateObject($normalizedData, $class, $context, $reflectionClass, $allowedAttributes);
+        $object = $this->instantiateComplexObject($normalizedData, $class, $context, $reflectionClass, $allowedAttributes, $format);
```

Comment:
Not sure to see what you mean, before it was simply failing so people could not used it. The method has been changed because there's a BC break in the method signature (we are adding `$format`), but the behaviour is the previous one.